extensions/elasticsearch/6.x/wazuh-template.json

```diff
@@ -1195,6 +1195,10 @@
                         }
                       }
                     },
+                    "condition": {

```

Comment:
Remove this field from the template, the `condition` field won't be shown as an alert field, same as `rule`.